cmd/kubeadm/app/cmd/init.go

```diff
@@ -338,6 +338,13 @@ func (i *Init) Run(out io.Writer) error {
 				return fmt.Errorf("error creating default audit policy %q [%v]", i.cfg.AuditPolicyConfiguration.Path, err)
 			}
 		}
+		// Verify the AuditWebhookConfigFile (verify it exists if it was passed in)
+		if i.cfg.AuditPolicyConfiguration.WebhookConfigPath != "" {
```

Comment:
The amount of audit initialization logic embedded in the init workflow is starting to get relevant. What about moving it in auditutil.InitAuditing or something?
@chuckha WDYT?

Reply:
I agree, this section is getting fairly somewhat unruly, but I also don't want to get in @hh's way. I think being able to enable webhook auditing is a pretty good feature. If @hh is up for it, the refactor would be fantastic, but if not, I would be ok with a ticket with priority/important-soon to refactor this section since it is indeed getting a bit large for this init section. It might even be a good help-wanted ticket for the kubecon session this week (maybe?).